docs/components/Form/Mobile.stories.tsx

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import { Content, Form, InputText } from "@jobber/components-native";
+
+export default {
+  title: "Components/Forms and Inputs/Form/Mobile",
+  component: Form,
+  parameters: {
+    viewMode: "story",
+    previewTabs: { code: { hidden: false } },
+  },
+} as ComponentMeta<typeof Form>;
+
+const BasicTemplate: ComponentStory<typeof Form> = args => (
+  <Form {...args}>
+    <Content>
+      <InputText
+        name="firstName"
+        placeholder="First name"
+        validations={{ required: "Please add a first name" }}
+      />
+      <InputText
+        name="lastName"
+        placeholder="Last name"
+        validations={{ required: "Please add a last name" }}
+      />
+      <InputText
+        name="nickName"
+        placeholder="Nick name"
+        validations={{ required: "Please add a nick name" }}
+      />
+    </Content>
+  </Form>
+);
+
+export const Basic = BasicTemplate.bind({});
+Basic.args = {
+  initialValues: {
+    firstName: "Greatest",
+    lastName: "Ever",
+    nickName: "",
+  },
+  onSubmit: value =>
+    new Promise(resolve => {
+      setTimeout(() => resolve(alert(JSON.stringify(value, undefined))), 1000);
+    }),
+};
+
+export const WithSecondaryActions = BasicTemplate.bind({});
```

Comment:
Clicking the `...` renders an error, which we know happens in ButtonGroup(?) or another component already.  We should maybe add `showNativeOnWebDisclaimer: true,` to the parameters at the top as well so that we show the disclaimer about it not working in web